### ISSUE TITLE
Need to support dangling spaces for a11y strings

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#779]
From a11y string conversion in https://github.com/phetsims/rosetta/issues/193.

We have some strings that purposefully end/begin in spaces. Currently this feature is only on a branch, but there should be an issue devoted towards fixing this. Right now the string plugin will trim the strings for us.

**Comment by zepumph** [comment]
The first thought I have here is perhaps we should make these purposeful spaces a special char, maybe a unicode space that `trim` doesn't strip out.

**Comment by zepumph** [comment]
It would be good to note https://github.com/phetsims/chipper/issues/619 in this discussion, since that issue is why `trim` calls were added. 

@samreid and I discussed and we expect that this constraint will be easy enough to get around by rethinking the template strings. After reading in https://github.com/phetsims/chipper/issues/619 I think it is not worth trying to support this. Over in https://github.com/phetsims/rosetta/issues/193 we will make sure that when a11y strings are moved into json files, that they are audited for dangling white space.
